facade/logstash_test.go

```diff
@@ -172,6 +267,46 @@ func TestConstructingFilterString(t *testing.T) {
 	}
 }
 
+func TestGetAuditLogSectionForNoservice(t *testing.T) {
+	services := []servicedefinition.ServiceDefinition{}
+	auditLogTypes := []string{}
+	auditLogSection := getAuditLogSection(services, &auditLogTypes)
+	if len(auditLogSection) > 0 {
+		t.Error(fmt.Sprintf("Audit Log Section should be empty but it is not %d", len(auditLogSection)))
+	}
+}
+
+func TestGetAuditLogSectionForServicesNotEnabledAudit(t *testing.T) {
+	services := make([]servicedefinition.ServiceDefinition, 1)
+	services[0] = getTestingService()
+	auditableTypes := []string{}
+	auditLogSection := getAuditLogSection(services, &auditableTypes)
+	if len(auditLogSection) != 0 {
+		t.Error(fmt.Printf("expected am empty auditLogSection , but found %d size", len(auditLogSection)))
```

Comment:
should be fmt.Sprintf()

Also, if this test fails in the future, it would speed up debugging if the error message also printed the value of `auditLogSection`